### ISSUE TITLE
0.4.0.4 is screwing PAW

### ISSUE THREAD

**Issue by Lisias** [net-lisias-ksp/KSP-Recall#75]
Someone is screwing some PAW entries on Flight!

Fellow Kerbonaut Krazy1 [reported](https://forum.kerbalspaceprogram.com/topic/192048-143/?do=findComment&comment=4376454):

> Here's what I'm seeing in flight: 
> ![image](https://github.com/net-lisias-ksp/KSP-Recall/assets/64334/543bd58e-4415-4440-b245-f4958f978b01)

But this is what I see on my rig (KSP 1.12.5 with the latest KSP-Recall):

### On Editor

![Screen Shot 2024-03-31 at 14 43 33](https://github.com/net-lisias-ksp/KSP-Recall/assets/64334/38184212-553f-408c-b398-2767c7edd2eb)

### On Flight

![Screen Shot 2024-03-31 at 14 44 02](https://github.com/net-lisias-ksp/KSP-Recall/assets/64334/d62c8f5a-af42-4e88-870a-ff96c78355b3)

**Comment by Lisias** [comment]
Oukey, Regression Test Fest.

I downgraded KSP-Recall (on KSP 1.12.5 with a minimal install set -- **WITHOUT** KSPCF, just to made it clear to my future self) to see what I get.

* Test Sesstions
    + Recall 0.4.0.3 worked fine. [KSP.log](https://github.com/net-lisias-ksp/KSP-Recall/files/14817384/KSP.log)
        - ![screenshot63](https://github.com/net-lisias-ksp/KSP-Recall/assets/64334/7698501b-2eb5-44ec-8b90-4a7ac02d4b1f)
    + Recall 0.4.0.4 Borked! [KSP.log](https://github.com/net-lisias-ksp/KSP-Recall/files/14817463/KSP.log)
        - ![Screen Shot 2024-03-31 at 16 42 03](https://github.com/net-lisias-ksp/KSP-Recall/assets/64334/a6785b7e-874f-4012-be20-25404e0e9cff)

Where "working fine" is the absence of the weird PAW entries reported by Krazy1 above.

**Comment by Lisias** [comment]
Oukey, my fault.

**Comment by Lisias** [comment]
When I wrote 0.4.0.3 (being specific, this commit https://github.com/net-lisias-ksp/KSP-Recall/commit/f8afb62c25f7045e44121dc6e8ce81e9bbde7309 ) I intentionally meant to tie the presence of the PAW's entries to the `DebugMode` on `KSPe`'s `Globals`.

One would need to manually edit the `Globals` file in order to see that PAWs, and I found no problem on cluttering the PAW with these entries on Debugging something. It was a way to visually check what's installed or not, and if the thing is active or not. But I never bothered to check **what would happen**, as I had something else on the pipeline and the focus was on **hidding** the PAWs for the common user.

Then this happened, and I realized these PAW entries were being used by some users, and stupidly just tied the PAWs visibility to a new option, normally set to `true`, and then what I had intended to happen only on `DebugMode` started to happen to everybody and the kitchen's sink. This naïve commit caused the problem: https://github.com/net-lisias-ksp/KSP-Recall/commit/8389132e21a53abd42aa5575ba790a1c266d54f5 - as dumb as I could get, to tell you the true.
